packages/application-generic/src/usecases/trigger-event/trigger-event.command.ts

```diff
@@ -19,8 +24,8 @@ export class TriggerEventCommand extends EnvironmentWithUserCommand {
   @IsDefined()
   overrides: Record<string, Record<string, unknown>>;
 
-  @IsDefined()
-  to: ISubscribersDefine[];
+  @IsOptional()
+  to?: ISubscribersDefine[];
```

Comment:
What's the reason behind making this optional?

Reply:
Good question, it is related to this thought i had 
https://github.com/novuhq/novu/pull/4526#discussion_r1368511918
I was not sure how to provide extra data to parse-event and trigger-event usecases that we need to execute broadcast trigger.

Reply:
Oh, wasn't sure what that meant.
I think we might need to explore to create specific command to each case sharing a base command.